doc/development-context-aware-applications/how-to-perform-geo-located-queries.md

```diff
@@ -2,10 +2,10 @@ One very powerful feature in Context Broker GE is the ability to perform
 geo-located queries. You can query entities located inside (or outside)
 a region defined by a circle or a polygon.  
  For example, to query for all the restaurants within 13 km of the
-Madrid city center (identified by GPS coordinates 40.418889, -3.691944)
+Victoria Gasteiz city center (identified by GPS coordinates 42.846718, -2.671635)
 a Context Consumer application will use the following query:
 
-    POST <cb_host>:<cb_port>/v1/queryContext
+    POST <cb_host>:<cb_port>/v1/contextEntities
```

Comment:
You are right. We had an issue with the REST client that we was using. Thank you for the feedback!